apps/coscrad-frontend/src/components/nav-menu/nav-menu.tsx

```diff
@@ -0,0 +1,98 @@
+import { CategorizableType } from '@coscrad/api-interfaces';
+import MenuRoundedIcon from '@mui/icons-material/MenuRounded';
+import { Box, IconButton, Menu } from '@mui/material';
+import { useContext, useState } from 'react';
+import { routes } from '../../app/routes/routes';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import AuthenticationButton from '../authentication-button/authentication-button';
+import { NavMenuItem } from './nav-menu-item';
+
+export type NavItemInfo = {
+    link: string;
+    label: string;
+};
+
+// TODO: We should have a NavBar container and presenter
```

Comment:
Bingo. I do not want to see styling in this file. It's job should be to dynamically create the array of menu items. It should defer to a  nav-menu-presenter for the styling.

Reply:
Done